packages/shared/src/hooks/auth/useOnboarding.tsx

```diff
@@ -33,7 +34,7 @@ export const useOnboarding = (): UseOnboarding => {
 
   return {
     shouldShowAuthBanner,
-    isOnboardingReady: isActionsFetched && isAuthReady,
+    isOnboardingReady: isActionsReady && isAuthReady,
```

Comment:
bit unclear what we want to do, since `isAuthReady` is specified both here and in above condition?

Reply:
Yeah, maybe the condition can be simplified, but yeah, the end result should be the same. Let me refactor this to be a 1-liner.